cleaned_data/modsbyPID/rftaart:10.xml

```diff
@@ -10,12 +10,7 @@
       <title>Sacrificing meaningful items during evacuation</title>
    </titleInfo>
    <originInfo>
-      <dateIssued>2021</dateIssued>
-      <dateIssued encoding="edtf" keyDate="yes">2021</dateIssued>
-      <publisher/>
-      <place>
-         <placeTerm valueURI="http://id.loc.gov/authorities/names/n79109786">Knoxville (Tenn.)</placeTerm>
-      </place>
+      <dateCreated encoding="edtf" keyDate="yes">2021</dateCreated>
```

Comment:
@mlhale7 Is this what you're referring to?

Reply:
Yes, what you have here is perfect.